### PR TITLE
Sync "Contribute" section with CONTRIBUTING.md

### LINKED ISSUE #143: Link to the CCLA PDF file is broken
The linked PDF file in the "Corporate Contributor License Agreement" section (``/doc/cla/SAP%20Corporate%20Contributor%20License%20Agreement%20(5-26-15).pdf``) in the file ``website/documentation/contribute/_index.md`` (published at https://gardener.cloud/documentation/contribute/) does not exist.

The file is available at https://github.com/cla-assistant/cla-assistant/blob/master/SAP%20Corporate%20Contributor%20License%20Agreement%20(5-26-15).pdf.

### PR DESCRIPTION
**Pull request by berendt**
Closes #143

### CODE REVIEW

File: website/documentation/contribute/_index.md [hunk 1]
```diff
@@ -92,4 +93,4 @@ We use GitHub issues to track bugs and enhancement requests and ZenHub for plann
 
 ## Security Release Process
 
-See [Security Release Process](https://github.com/gardener/documentation/blob/master/security-release-process.md)
\ No newline at end of file
+See [Security Release Process](https://github.com/gardener/documentation/blob/master/security-release-process.md)
```
Comment (berendt) [inline website/documentation/contribute/_index.md:96]: Added a missing \n here.

**Comment by msohn** [comment]
@g-pavlov AFAIK this file is generated from https://github.com/gardener/documentation/blob/master/CONTRIBUTING.md
Can you please check why it wasn't re-generated ?

**Comment by berendt** [comment]
I think this is the best way. The single source of trust is the contributor guide on gardener.cloud.  All CONTRIBUTING.md files refer to this guide (this is not yet the case, needs to be changed in various repositories)

**Comment by g-pavlov** [comment]
I explained in more detail the reason in #143. The source of truth is here: https://github.com/gardener/documentation/blob/master/CONTRIBUTING.md . We were copying this in the documentation bundle automatically upon build, but had to cease that for a while. This document doesn't change very often if at all and we assumed that will not cause trouble but unfortunately that is just hat happened :-) We'll be fixing that shortly. In the mean time the PR can be closed as this _index.md is auto-generated from the source file mentioned above.

**Pull request closed without merging**